sql/core/src/test/resources/sql-tests/results/show-tables.sql.out

```diff
@@ -120,46 +120,35 @@ show_t3
 -- !query
 SHOW TABLE EXTENDED LIKE 'show_t*'
 -- !query schema
-struct<namespace:string,tableName:string,isTemporary:boolean,information:string>
+struct<namespace:string,tableName:string,isTemporary:boolean,information:map<string,string>>
 -- !query output
-	show_t3	true	Table: show_t3
+Database:showdb
+Table:show_t1
 Created Time [not included in comparison]
 Last Access [not included in comparison]
 Created By [not included in comparison]
-Type: VIEW
-Table Properties: []
-Schema: root
- |-- e: integer (nullable = true)
-
-
-showdb	show_t1	false	Database: showdb
-Table: show_t1
+Type:MANAGED
+Provider:parquet
+Location [not included in comparison]/{warehouse_dir}/showdb.db/show_t1
+Partition Provider:Catalog
+Partition Columns:[`c`, `d`]
+Schema:struct<a:string,b:int,c:string,d:string>
+Database:showdb
```

Comment:
need a line-separator before this line like hive?

Reply:
Yes. fixed it.